pkg/providers/aws/source.go

```diff
@@ -0,0 +1,84 @@
+package amazon
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/re-cinq/aether/pkg/config"
+	v1 "github.com/re-cinq/aether/pkg/types/v1"
+)
+
+// Source is a configured Amazon source that adheres to Aethers source
+// interface
+type Source struct {
+	// Amazon Client
+	*Client
+
+	// The Project that Client is setup for
+	Project *string
+
+	// AWS doesn't use project, but instead is
+	// separated by region
+	Region string
+
+	// Teardown functionality
+	Shutdown func()
```

Comment:
Do we need this for AWS?

Reply:
good point, I was thinking it had to match the interface exactly